strapi/strapi
# DynamicZone - extend FriendlyName with MainValue

...like RepeatableComponent

This commit fixes #12499 



### What does it do?

Show the MainValue of the Components of a DynamicZone in the AccordionToggle.

### Why is it needed?

To be able to distingish the Dzone components without unfold them.

### How to test it?

- Create a Component
- Create a Collection
  - add DynamicZone
    - add the Component
- Edit the Collection
  - edit DZone
    - add more Component instance
    - edit the mainField
    - check the AccordionToggle changes

Sample:
![strapi_dzone_main_value](https://user-images.githubusercontent.com/912200/153741957-15038a05-77d9-480b-ac55-53311fd372a1.jpg)


### Related issue(s)/PR(s)

Related issue: #12499
